<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>869</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vehicle Report Shower</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>1061</width>
      <height>774</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Select Model</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="yolo">
           <property name="text">
            <string>YOLOV3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="ssd">
           <property name="text">
            <string>SSD</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item alignment="Qt::AlignHCenter|Qt::AlignTop">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Vehicle Report</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Upload Video</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="report">
       <property name="text">
        <string>Show Report</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="debugTextBrowser">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="original_video">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>300</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="difference_video">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>300</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>browseSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1037</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>784</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>returnPressedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>529</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>775</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yolo</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>yoloSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>846</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ssd</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>ssdSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>817</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>report</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>reportSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>831</x>
     <y>461</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browseSlot()</slot>
  <slot>slot1()</slot>
  <slot>returnPressedSlot()</slot>
  <slot>yoloSlot()</slot>
  <slot>ssdSlot()</slot>
  <slot>reportSlot()</slot>
  <slot>videoShowSlot()</slot>
 </slots>
</ui>
